Assets/Scripts/Models/World.cs

```diff
@@ -166,7 +166,7 @@ void SetupWorld(int width, int height)
                 tiles[x, y].Room = GetOutsideRoom(); // Rooms 0 is always going to be outside, and that is our default room
             }
         }
-
+        new NeedActions ();
```

Comment:
@Mizipzor The NeedActions instantiator creates a static reference to itself, keeping a reference in World is unnecessary.